base/shell/cmd/resource.h

```diff
@@ -213,6 +213,7 @@
 #define STRING_RMDIR_HELP2                 724
 #define STRING_MD_ERROR                    725
 #define STRING_MD_ERROR2                   726
+#define STRING_DATE_CURRENT_DATE           727
```

Comment:
Please rename it as `STRING_DATE_NOW` to follow a similar name as the existing [`STRING_TIME_NOW`](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=STRING_TIME_NOW)

Reply:
Fixed.